### PR TITLE
Adding Plazza Fridge

### PR DESCRIPTION
**Pull request by 31Nathan**
Add Plazza Fridge to Kitchen Class
Fill it every refill time on the thread loop

### CODE REVIEW

File: src/Core/Kitchen/Kitchen.cpp [hunk 1]
```diff
@@ -57,3 +61,18 @@ void Kitchen::stop()
     }
     _brigade.clear();
 }
+
+void Kitchen::_fillFridge(const std::size_t &timeToFill, Kitchen &obj)
+{
+    for (size_t x = 0; x < IngredientNumber; ++x) {
```
Comment (osvegn) [inline src/Core/Kitchen/Kitchen.cpp:67]: use std::size_t

**Review by osvegn — CHANGES_REQUESTED** [review]
Need minor change

File: src/Core/Kitchen/Kitchen.cpp (hunk 1 above)
Comment (Cavonstavant) [reply src/Core/Kitchen/Kitchen.cpp:67]: Why @osvegn ?

Comment (osvegn) [reply src/Core/Kitchen/Kitchen.cpp:67]: size_t -> C
std::size_t -> C++
?

**Pull request merged**